spec/mongo/error/operation_failure_spec.rb

```diff
@@ -148,6 +148,52 @@
         end
       end
     end
+
+    context 'when there is a non-resumable label' do
+      context 'getMore response' do
+        let(:error) { Mongo::Error::OperationFailure.new('no message',
+          Mongo::Operation::GetMore::Result.new([]),
+          :code => 91, :code_name => 'ShutdownInProgress',
+          :labels => ['NonResumableChangeStreamError']) }
+
+        it 'returns false' do
+          expect(error.change_stream_resumable?).to eql(false)
+        end
+      end
+
+      context 'not a getMore response' do
+        let(:error) { Mongo::Error::OperationFailure.new('no message', nil,
+          :code => 91, :code_name => 'ShutdownInProgress',
+          :labels => ['NonResumableChangeStreamError']) }
+
+        it 'returns false' do
+          expect(error.change_stream_resumable?).to eql(false)
+        end
+      end
+    end
+
+    context 'when there is another label' do
+      context 'getMore response' do
+        let(:error) { Mongo::Error::OperationFailure.new('no message',
+          Mongo::Operation::GetMore::Result.new([]),
+          :code => 91, :code_name => 'ShutdownInProgress',
+          :labels => [ Mongo::Error::TRANSIENT_TRANSACTION_ERROR_LABEL ]) }
+
+        it 'returns true' do
+          expect(error.change_stream_resumable?).to eql(true)
+        end
+      end
+
+      context 'not a getMore response' do
+        let(:error) { Mongo::Error::OperationFailure.new('no message', nil,
+          :code => 91, :code_name => 'ShutdownInProgress',
+          :labels => [ Mongo::Error::TRANSIENT_TRANSACTION_ERROR_LABEL ]) }
+
+        it 'returns false' do
+          expect(error.change_stream_resumable?).to eql(false)
+        end
+      end
+    end
```

Comment:
Can a test also be added which tests what happens when there is no label?

Reply:
The other tests for `change_stream_resumable?` in this file do not have labels. I didn't want to be repetitive, but is it better to explicitly include similar tests?

Reply:
My mistake, everything here is already correct.